_config.yml

```diff
@@ -56,14 +56,14 @@ downloads:
       zip: 2ad4eaabfd92d627baffc6c971e4b8987b38c06baf42dc2fc2e05131095499e7
   stable_snapshot:
     url:
-      bz2: https://ftp.ruby-lang.org/pub/ruby/stable-snapshot.tar.bz2
```

Comment:
This may be on purpose since (as I understand it) ftp.ruby-lang.org is the origin server used for cache.ruby-lang.org, and since these and the ones below are snapshots, not sure the CDN makes sense, especially since they change so often.